src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const BooksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.bookItems = state.bookItems.filter((item) => item.id !== bookId);
```

Comment:
- [ ] Good job working on "Bookstore: add reducers and actions" 😃, however, currently, I'm not able to `add` and `remove` from your `books` table 😭. Kindly make sure you fixed it to make your action functional 💪.